openedx/core/djangoapps/course_groups/tests/test_cohorts.py

```diff
@@ -253,8 +253,8 @@ def test_get_cohort(self):
         )
 
     @ddt.data(
-        (True, 2),
-        (False, 6),
+        (True, 8), # 6 Django savepoints & 2 cohort SELECTs
+        (False, 12), # 6 Django savepoints & 6 cohort SELECTs
```

Comment:
@nedbat @symbolist The extra savepoints were caused by the test calling a method that had a `@transaction.atomic` decorator:

https://github.com/edx/edx-platform/blob/django-upgrade/1.8/openedx/core/djangoapps/course_groups/cohorts.py#L140

Even though the method performed no SQL DB calls due to caching, it still performed a SAVEPOINT/RELEASE SAVEPOINT.